planner/memo/pattern.go

```diff
@@ -120,26 +120,27 @@ func (o Operand) Match(t Operand) bool {
 
 // Pattern defines the Match pattern for a rule.
 // It describes a piece of logical expression.
-// It's a tree-like structure and each node in the tree is an Operand.
+// It's a tree-like structure and each node in the tree is an Operand and an EngineType.
```

Comment:
nit: how about reformat the comment to:
```go
// Pattern defines the match pattern for a rule. It's a tree-like structure
// which is a piece of a logical expression. Each node in the Pattern tree is
// defined by a Operand and EngineType pair.
```

The width of each line in the comment is limit to at most 80.

Reply:
Sure, thanks.